cura/BuildVolume.py

```diff
@@ -746,7 +746,8 @@ def _updateDisallowedAreas(self):
                                 break
                         if prime_tower_collision: #Already found a collision.
                             break
-                        if ExtruderManager.getInstance().getResolveOrValue("prime_tower_brim_enable"):
+                        if (ExtruderManager.getInstance().getResolveOrValue("prime_tower_brim_enable") and
+                            ExtruderManager.getInstance().getResolveOrValue("adhesion_type") != 'raft'):
```

Comment:
```suggestion
                            ExtruderManager.getInstance().getResolveOrValue("adhesion_type") != "raft":
```
More nitpciking: Use " instead of '